README.md

```diff
@@ -71,16 +91,16 @@ At any point, create a branch for your contribution.
 We are not strict about branch names.
 
 ```shell
-git checkout -b add/klingon-language
+git checkout -b add/partial-hydration-typo-fix
 ```
 
 That’s it.
 As you [open a pull request](https://github.com/withastro/astro/compare), please include a clear title and description.
 
 ```markdown
-# Add Klingon language to Getting Started page
+# Fixes a typo on Partial Hydration page
 
-This adds the Klingon language and also updates the sidebar and language selection components.
+This fixes a typo in the page on partial hydration, and additionally removes an unnecessary extra space.
 ```
```

Comment:
Maybe we can get away without this example here now that PRs are pre-populated with a template?

Reply:
I like it even better!